gui/app/services/chatMessagesService.js

```diff
@@ -122,6 +122,13 @@
                 }
             };
 
+            service.highlightMessage = (rawText) => {
+                backendCommunicator.fireEvent("highlight-message", {
+                    action: "highlightMessage",
+                    messageText: rawText
+                });
+            };
```

Comment:
We'll need to update this function to the following:
```js
service.highlightMessage = (username, rawText) => {
    backendCommunicator.fireEvent("highlight-message", {
        username: username,
        chatMessage: rawText
    });
};
```
What I changed:
- Adds `username` as the first arg so we can pass it along in the event data. You're already calling this properly in that switch statement (`chatMessagesService.highlightMessage(userName, rawText);`)
- Added `username` property to the event data object.
- Removed the `action` property as its not needed
- updated `messageText` to `chatMessage` to match what the `$chatMessage` var is expecting